src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/ImplementISerializableCorrectly.cs

```diff
@@ -1,65 +0,0 @@
-// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the MIT license.  See License.txt in the project root for license information.
-
-using System.Collections.Immutable;
-using Microsoft.CodeAnalysis;
-using Microsoft.CodeAnalysis.Diagnostics;
-
-namespace Microsoft.NetCore.Analyzers.Runtime
-{
-    /// <summary>
-    /// CA2240: Implement ISerializable correctly
-    /// </summary>
-    public abstract class ImplementISerializableCorrectlyAnalyzer : DiagnosticAnalyzer
-    {
-        internal const string RuleId = "CA2240";
-
-        /*private static readonly LocalizableString s_localizableTitle = CreateLocalizableResourceString(nameof(ImplementISerializableCorrectlyTitle));
-        private static readonly LocalizableString s_localizableDescription = CreateLocalizableResourceString(nameof(ImplementISerializableCorrectlyDescription));
-
-        internal static readonly DiagnosticDescriptor DefaultRule = DiagnosticDescriptorHelper.Create(
-            RuleId,
-            s_localizableTitle,
-            CreateLocalizableResourceString(nameof(ImplementISerializableCorrectlyMessageDefault)),
-            DiagnosticCategory.Usage,
-            RuleLevel.Disabled,
-            description: s_localizableDescription,
-            isPortedFxCopRule: true,
-            isDataflowRule: false,
-            isEnabledByDefaultInFxCopAnalyzers: false);
-
-        internal static readonly DiagnosticDescriptor MakeVisibleRule = DiagnosticDescriptorHelper.Create(
-            RuleId,
-            s_localizableTitle,
-            CreateLocalizableResourceString(nameof(ImplementISerializableCorrectlyMessageMakeVisible)),
-            DiagnosticCategory.Usage,
-            RuleLevel.Disabled,
-            description: s_localizableDescription,
-            isPortedFxCopRule: true,
-            isDataflowRule: false,
-            isEnabledByDefaultInFxCopAnalyzers: false);
-
-        internal static readonly DiagnosticDescriptor MakeOverridableRule = DiagnosticDescriptorHelper.Create(
-            RuleId,
-            s_localizableTitle,
-            CreateLocalizableResourceString(nameof(ImplementISerializableCorrectlyMessageMakeOverridable)),
-            DiagnosticCategory.Usage,
-            RuleLevel.Disabled,
-            description: s_localizableDescription,
-            isPortedFxCopRule: true,
-            isDataflowRule: false,
-            isEnabledByDefaultInFxCopAnalyzers: false);*/
-
-        public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics { get; } = ImmutableArray<DiagnosticDescriptor>.Empty;
```

Comment:
This and `ImplementSerializationMethodsCorrectly.cs` are not implemented and has empty C#, VB overloads and C#/VB Fixers with overloads, kind of unrelated, but has no effect, I can remove them separately if needed